temporalcli/commandsgen/commands.yml

```
@@ -605,6 +605,179 @@ commands:
         description: Reason for terminating the batch job.
         required: true
 
+  - name: temporal worker
+    summary: Read or update Worker state
+    description: |
+      Modify or read state associated with a Worker, for example,
+      using Worker Deployments commands:
+
+      ```
+      temporal worker deployment
+      ```
+    option-sets:
+      - client
+    docs:
+      description-header: >-
+         Learn how to read or modify state associated with a Worker,
+         such as Worker Deployments.
+      keywords:
+        - worker
+        - worker deployment
+
+  - name: temporal worker deployment
+    summary: Describe, list, and operate on Worker Deployments
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Deployment commands perform operations on Worker Deployments:
+
+      ```
+      temporal worker deployment [command] [options]
+      ```
+
+      For example:
+
+      ```
+      temporal worker deployment list
+      ```
+    docs:
+      description-header: >-
+        Temporal Deployment commands enable operations on Worker Deployments,
+        such as describe, list, set-current, and get-current, simplifying
+        versioning and management of workers.
+      keywords:
+        - worker deployment
+        - worker deployment describe
+        - worker deployment list
+        - worker deployment get-current
+        - worker deployment set-current
+
+  - name: temporal worker deployment describe
+    summary: Show properties of a Worker Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Describes properties of a Worker Deployment, such as whether it is
+      current, the non-empty list of its task queues, custom metadata if
+      present, and reachability status when requested.
+
+      ```
+      temporal worker deployment describe [options]
+      ```
+
+      For example, to also include reachability information:
+
+      ```
+      temporal worker deployment describe \
+          --series-name YourDeploymentSeriesName \
+          --build-id YourDeploymentBuildId \
+          --report-reachability
+      ```
+    option-sets:
+      - deployment-reference
+    options:
+      - name: report-reachability
+        type: bool
+        description: |
+          Flag to include reachability information of a Worker Deployment.
```

You don't need to say it's a flag. That's obvious in context:

```suggestion
          Include reachability information of a Worker Deployment.
```

---

Thanks, much better